packages/adapter-cloudflare/package.json

```diff
@@ -56,5 +56,8 @@
 	},
 	"publishConfig": {
```

Comment:
you're on crack, github. this code doesn't exist: https://github.com/sveltejs/kit/blob/2b19df3855b13737399a7173f25e4fecbd7f689a/packages/adapter-cloudflare/package.json#L57

wtf is happening here??

Reply:
https://github.com/sveltejs/kit/commit/2b19df3855b13737399a7173f25e4fecbd7f689a#diff-1ac6ebb9247195e0dfa0f18975077643b0e0743c80cd8588b0084f5252cecec0 Changesets removed this when it bumped the version number for the release.